src/main/resources/resources/logisim/strings/gui/gui.properties

```diff
@@ -102,7 +102,7 @@ BusFormat = Format
 # generic/AttributeSetTableModel.java
 #
 attributeChangeInvalidError = Attribute unchanged because request is invalid
-FPGA_Supported = FPGA supported:
+FPGA_Supported = FPGA supported
```

Comment:
Currently, this shows in the attributes table as "FPGA supported: Supported".

I would suggest to change this to "FPGA support: yes" (or "no"). Of course, this also has impact on the other localisation strings and the implementation of the attribute table above (`src/main/java/com/cburch/logisim/fpga/gui/HdlColorRenderer.java`).

Reply:
Yes, I was thinking the same. But I will go for this in separate PR